content/backmatter.tex

```diff
@@ -501,6 +501,12 @@ \section{Version 1.4}
 \item Added the \FUNC{shmem\_calloc} function.
 \\ See Section \ref{subsec:shmem_calloc}.
 %
+\item Clarified description for \CONST{SHMEM\_VENDOR\_STRING}.
+\\See Section \ref{subsec:library_constants}.
+%
+\item Clarified API description for \FUNC{shmem\_info\_get\_name}.
+\\See Section \ref{subsec:shmem_info_get_name}.
+%
 \end{itemize}
 
 \section{Version 1.3}
```

Comment:
Changes since reading
- New changelog entry for changes to `SHMEM_VENDOR_STRING`
- Fixed --> Clarified